docs/lib/graphqlapi/fragments/android/getting-started/40_codegen.md

```diff
@@ -6,4 +6,4 @@ To generate the `Todo` model, change directories to your project folder and **ex
 amplify codegen models
 ```
 
-The generated files will be under the `app/src/main/java/com/amplifyframework.datastore.generated.model` directory. 
+The generated files will be under the `app/src/main/java/com/amplifyframework.datastore.generated.model` directory. It is strongly advised not to put any hand written code in this directory as it gets re-generated each time codegen is run. 
```

Comment:
Could this be generalized to anything under `amplify/amplifyframework.*.generated.*` directories?

Reply:
We do not generate anything other than `models` in the `generated` folder, would be nice to keep the buffer and generalize it. Made this change for iOS and android.